docs/API.md

```diff
@@ -214,9 +214,9 @@ This is useful in combination with another helper that expects a higher-order co
 // has all the props it needs
 const spinnerWhileLoading = (hasLoaded, BaseComponent) => branch(
   hasLoaded,
-  BaseComponent,
+  renderComponent(BaseComponent),
   renderComponent(Spinner) // <Spinner> is a React component
-);
+)(renderNothing());
```

Comment:
You actually don't need this line. `spinnerWhileLoading` is a higher-order component, not a component.